tests/storage/cdi_upload/test_upload_virtctl.py

```
@@ -281,13 +282,18 @@ def empty_pvc(
     storage_class_matrix__module__,
     storage_class_name_scope_module,
 ):
+    min_pvc_size = get_storage_profile_minimum_supported_pvc_size(storage_class_name=storage_class_name_scope_module)
+    pvc_size = min_pvc_size or DEFAULT_DV_SIZE
+    size_source = "storage profile" if min_pvc_size else "default"
+    LOGGER.info(f"Using PVC size '{pvc_size}' ({size_source}) for storage class '{storage_class_name_scope_module}'")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)